common/config/azure-pipelines/npm-publish-rush-pr.yaml

```diff
@@ -0,0 +1,41 @@
+pool:
+  vmImage: 'ubuntu-latest'
```

Comment:
@theJiawen seems like your branch is bringing in changes from the `release/rush-pr3481` branch that are not part of the `main` branch.

Reply:
Oops, fixed it.